### PR TITLE
docs(root): remove node 16

### PR DESCRIPTION
**Pull request by zahin-mohammad**
This was missed as part of https://github.com/BitGo/BitGoJS/pull/4119
TICKET: WP-1100

### CODE REVIEW

**Review by gianchandania — DISMISSED** [review]

File: DEVELOPERS.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Requirements
 
 - yarn version 1 (classic)
-- node version >=16 and <19
+- node version >=18 and <20
```
Comment (andrew-scott-fischer) [inline DEVELOPERS.md:4]:
```suggestion
- node version >=18 and <21
```

**Review by andrew-scott-fischer — DISMISSED** [review]
should be `<21`

File: DEVELOPERS.md (hunk 1 above)
Comment (andrew-scott-fischer) [reply DEVELOPERS.md:4]: we support `20`, but not `21`

**Review by andrew-scott-fischer — COMMENTED** [review]
I pushed the fix.

**Review by aarvay — APPROVED** [review]

**Pull request merged**